StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1633UnitTests.cs

```diff
@@ -100,6 +103,19 @@ namespace Foo
 
             var expectedDiagnostic = this.CSharpDiagnostic(FileHeaderAnalyzers.SA1633DescriptorMissing).WithLocation(1, 1);
             await this.VerifyCSharpDiagnosticAsync(testCode, expectedDiagnostic, CancellationToken.None).ConfigureAwait(false);
+
+            var fixCode = @"// <copyright file=""Test0.cs"" company=""FooCorp"">
```

Comment:
:bulb: Can you name all of these `fixedCode`, and move them up to be defined on the line following `testCode`?